### PR TITLE
Make some fields optional and add missing fields

### PR DESCRIPTION
**Pull request by tahini**
This PR makes the types reflect the specification, by setting some fields as optional and adding the missing fields.

### CODE REVIEW

**Comment by tahini** [comment]
It partly fixes #5. If it is desirable, I can update the other objects the same way.

File: src/static.d.ts [hunk 1]
```diff
@@ -25,54 +25,70 @@ export const enum TransferType {
...
   route_type: VehicleType;
-  route_url: string;
-  route_color: string;
-  route_text_color: string;
+  route_url?: string;
+  route_color?: string;
+  route_text_color?: string;
+  route_sort_order?: number;
+  continuous_pickup?: 0 | 1 | 2 | 3 | '';
+  continuous_drop_off?: 0 | 1 | 2 | 3 | '';
```
Comment (k-yle) [inline src/static.d.ts]: could we use an enum for these two please, something like `{CONTINUOUS=0, NON_CONTINUOUS, MUST_CONTACT_AGENCY, MUST_CONTACT_DRIVER }` to be consistent with `drop_off_type`

File: src/static.d.ts [hunk 2]
```diff
@@ -25,54 +25,70 @@ export const enum TransferType {
...
+  stop_name?: string;
+  stop_desc?: string;
+  stop_lat?: number;
+  stop_lon?: number;
+  zone_id?: string;
+  stop_url?: string;
+  location_type?: LocationType;
+  parent_station?: string;
+  stop_timezone?: string;
+  wheelchair_boarding?: 0 | 1 | 2;
```
Comment (k-yle) [inline src/static.d.ts]: ideally we would use an enum here too, maybe: `{ UNKNOWN=0, ACCESSIBLE, NOT_ACCESSIBLE }`

**Review by k-yle — CHANGES_REQUESTED** [review]
thanks for this PR. I think it's a logical change, I've left some minor comments. Thanks!

File: src/static.d.ts (hunk 1 above)
Comment (tahini) [reply src/static.d.ts]: Done

File: src/static.d.ts (hunk 2 above)
Comment (tahini) [reply src/static.d.ts]: Done

**Review by tahini — COMMENTED** [review]
Thanks for the quick response!

**Pull request merged**